ci-operator/step-registry/aws/provision/vpc/shared/aws-provision-vpc-shared-commands.sh

```diff
@@ -13,7 +13,10 @@ TAGS="Key=expirationDate,Value=${EXPIRATION_DATE}"
 
 REGION="${LEASED_RESOURCE}"
 
-curl -L https://raw.githubusercontent.com/openshift/installer/master/upi/aws/cloudformation/01_vpc.yaml -o /tmp/01_vpc.yaml
+vpc_tpl="/tmp/vpc_tpl.yaml"
+
+curl -L https://raw.githubusercontent.com/yunjiang29/ocp-test-data/main/upi_on_aws-cloudformation-templates/01_vpc.yaml -o "${vpc_tpl}" &
+wait "$!"
```

Comment:
The curl command need `wait` ?

Reply:
I thought that the `curl` command does not require `wait`,  but I got a following error in [this test](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/29050/rehearse-29050-periodic-ci-openshift-verification-tests-master-installer-rehearse-installer-rehearse-aws-qe-upi-disconnected-private/1532232783676575744), the output `/tmp/01_vpc.yaml` is missing:
```
INFO[[20](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/29050/rehearse-29050-periodic-ci-openshift-verification-tests-master-installer-rehearse-installer-rehearse-aws-qe-upi-disconnected-private/1532232783676575744#1:build-log.txt%3A20)22-06-02T05:31:53Z] Logs for container test in pod installer-rehearse-aws-qe-upi-disconnected-private-aws-provision-vpc-disconnected: 
INFO[2022-06-02T05:31:53Z]   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 6113k  100 6113k    0     0  18.8M      0 --:--:-- --:--:-- --:--:-- 18.8M
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  9498  100  9498    0     0  42899      0 --:--:-- --:--:-- --:--:-- 42783
Error parsing parameter '--template-body': Unable to load paramfile file:///tmp/01_vpc.yaml: [Errno 2] No such file or directory: '/tmp/01_vpc.yaml'
```
After adding `wait`, it always get passed.

Reply:
okay. Get it.